doc/rst/developer/chips/15.rst

```diff
@@ -232,7 +232,7 @@ related issues (mentioned later on).
 
 Another reason why we decided not to move with our approach based on
 disabling dead code elimination was to avoid having a large
-departure from how the same file would appear on master. Which would
+departure from how the same file would appear on main. Which would
```

Comment:
I wonder whether "on main" is somewhat ambiguous without more context.  Maybe:

* "on 'main'" to indicate that 'main' isn't just the English word, but something special?
* "on the 'main' branch" (or without quotes now that it's not serving as a special identifier as much)
* "on the 'main' development branch" (ditto)